sdks/python/apache_beam/transforms/core.py

```diff
@@ -851,9 +851,8 @@ def with_outputs(self, *tags, **main_kw):
     return _MultiParDo(self, tags, main_tag)
 
   def _pardo_fn_data(self):
-    si_tags_and_types = None
     windowing = None
-    return self.fn, self.args, self.kwargs, si_tags_and_types, windowing
```

Comment:
This is is the cause of the 

```
  File "/usr/local/google/home/lcwik/git/beam/sdks/python/apache_beam/runners/worker/operations.py", line 360, in start
    pickler.loads(self.spec.serialized_fn))
ValueError: need more than 4 values to unpack
```

failure. I don't remember if we completely removed its use in the legacy worker, but if so, we can probably remove it there too rather than re-introduce it here.

Reply:
I was able to remove this in the few places it was referenced.